conf/test_full.config

```diff
@@ -35,10 +35,10 @@ params {
 
     // Panelprep optional args
     remove_samples = "NA12878,NA12891,NA12892"
-    normalize      = true
-    compute_freq   = true
-    phase          = true
+    normalize      = false
```

Comment:
I am not sure if this will work without normalizing. Test with full chromosome test before full test.

Reply:
Effectively to remove the samples from the panel reference we need the normalisation step.
I will add it back.